src/models/unit_tests/piranha_arguments_test.rs

```diff
@@ -11,6 +11,19 @@
  limitations under the License.
 */
 
+/*
+ Copyright (c) 2023 Uber Technologies, Inc.
+
+ <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ except in compliance with the License. You may obtain a copy of the License at
+ <p>http://www.apache.org/licenses/LICENSE-2.0
+
+ <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ express or implied. See the License for the specific language governing permissions and
+ limitations under the License.
+*/
```

Comment:
Now there are two license headers here, with different dates...

Reply:
I know why :| Its the pre-commit hook. It matches for license with 2022, since it doesnt find it adds one :|

Reply:
I think configuring that pre-commit hook is tricky. It doesnt have an or option. We might have to bump all the licenses to 2023 or remove the hook